src/test/resources/pages/InstructorCourseDetailsDisplayRenameTeamModal.html

```diff
@@ -0,0 +1,634 @@
+            Edit
+          </a>
+          <a class="btn btn-default btn-xs student-delete-for-test" data-original-title="Delete the student and the corresponding submissions from the course" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseStudentDelete?courseid=CCDetailsUiT.CS2103&studentemail=danny.e.tmms%40gmail.tmt&user=CCDetailsUiT.instr2" onclick="return toggleDeleteStudentConfirmation('CCDetailsUiT.CS2103','Danny Engrid')" title="">
+            Delete
+          </a>
+          <a class="btn btn-default btn-xs student-records-for-test" data-original-title="View all data about this student" data-placement="top" data-toggle="tooltip" href="/page/instructorStudentRecordsPage?courseid=CCDetailsUiT.CS2103&studentemail=danny.e.tmms%40gmail.tmt&user=CCDetailsUiT.instr2" target="_blank" title="">
+            All Records
+          </a>
+          <div class="btn-group">
+            <a class="btn btn-default btn-xs cursor-default" data-original-title="Give a comment for this student, his/her team/section" data-placement="top" data-toggle="tooltip" href="javascript:;" title="">
+              Add Comment
+            </a>
+            <a class="btn btn-default btn-xs dropdown-toggle" data-toggle="dropdown" href="javascript:;">
+              <span class="caret">
+              </span>
+              <span class="sr-only">
+                Add comments
+              </span>
+            </a>
+            <ul aria-labelledby="dLabel" class="dropdown-menu align-left" role="menu">
+              <li role="presentation">
+                <a href="/page/instructorCourseStudentDetailsPage?courseid=CCDetailsUiT.CS2103&studentemail=danny.e.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&addComment=student" role="menuitem" tabindex="-1" target="_blank">
+                  Comment on student: Danny Engrid
+                </a>
+              </li>
+              <li role="presentation">
+                <a href="/page/instructorCourseStudentDetailsPage?courseid=CCDetailsUiT.CS2103&studentemail=danny.e.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&addComment=team" role="menuitem" tabindex="-1" target="_blank">
+                  Comment on team: Team 3
+                </a>
+              </li>
+              <li role="presentation">
+                <a href="/page/instructorCourseStudentDetailsPage?courseid=CCDetailsUiT.CS2103&studentemail=danny.e.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&addComment=section" role="menuitem" tabindex="-1" target="_blank">
+                  Comment on section: Tut Group 2
+                </a>
+              </li>
+            </ul>
+          </div>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+  <br>
+  <br>
+  <br>
+  <div aria-hidden="false" aria-labelledby="rename-team-modal-label" class="modal fade in" id="rename-team-modal" role="dialog" style="display: block;" tabindex="-1">
```

Comment:
@JosephineKwa is the modal the only important thing being tested here? In that case, use `verifyHtmlPart` rather than `verifyHtmlMainContent`.